client/js/components/common/audio_upload.jsx

```diff
@@ -0,0 +1,54 @@
+import React     from "react";
+import Recorder  from "./recorder";
+
+class AudioUpload extends React.Component {
+
+  static propTypes = {
+    selectAnswer: React.PropTypes.func,
+  };
+
+  constructor(){
+    super();
+    this.state = {
+      recorder: "stop",
```

Comment:
It looks like you're doing only using this for if you're recording, and not actually using the string. Just convert it to a boolean called isRecording if that's true.